cogs/public.py

```diff
@@ -32,7 +39,7 @@ async def ping(self, ctx: commands.Context):
         """
         Measure latency.
         """
-        raise NotImplementedError('Command requires implementation and permission set-up.')
+        await ctx.reply(embed=discord.Embed(description=f'I don\'t see how this will help you, but my ping is `{round(self.bot.latency * 1000)}ms`',color=discord.Color.orange()))
```

Comment:
I think in this context, its better to use double quotation marks for the string, so that you don't have to escape the single quote.